openapi.yaml

```diff
@@ -132,6 +172,39 @@ components:
       properties:
         asyncapi:
           $ref: '#/components/schemas/AsyncAPIDocument'
+    ConvertRequest:
+      type: object
+      required:
+        - asyncapi
+      properties:
+        asyncapi:
+          $ref: '#/components/schemas/AsyncAPIDocument'
+        version:
+          $ref: '#/components/schemas/SpecVersions'
+        language:
+          type: string
+          description: Language to be converted to.
+          enum:
+            - 'yaml'
+            - 'json'
+    SpecVersions:
+      type: string
+      description: Valid spec versions for the AsyncAPI document.
+      enum:
+        - '1.0.0'
+        - '1.1.0'
+        - '1.2.0'
+        - '2.0.0-rc1'
+        - '2.0.0-rc2'
+        - '2.0.0'
+        - '2.1.0'
+        - '2.2.0'
+        - 'latest'
+    ConvertResponse:
+      type: object
+      properties:
+        asyncapi:
```

Comment:
I wonder if we should rename that field to the `converted` but it's also ok :)

Reply:
`converted` sounds more explicit, I'll change it 👍

Reply:
Better stay with `asyncapi` 😃  let's be consistent in every place.